doi-requests-commons/src/main/java/no/unit/nva/doi/requests/service/impl/DynamoDBDoiRequestsService.java

```diff
@@ -120,24 +120,19 @@ public DynamoDBDoiRequestsService(AmazonDynamoDB client,
     @Override
     public Optional<DoiRequestSummary> fetchDoiRequest(UUID publicationId) throws NotFoundException {
         Publication publication = fetchPublication(publicationId);
-        return Optional.ofNullable(DoiRequestSummary.fromPublication(publication));
+        return Optional.of(DoiRequestSummary.fromPublication(publication));
     }
 
     @Override
     public void createDoiRequest(CreateDoiRequest createDoiRequest, String username)
```

Comment:
I believe we might want to refactor into `createDoiRequest` in `Publication` so we are telling the `Publication` object to create the doi request and it does the job and validation. 

Something that conflicts with my brain that it is `DynamoDBDoiRequestsService` doing the validation here.

Reply:
I see the point of moving the business logic closer to the datamodel classes. But we should agree on a plan for this. We cannot just start moving functionality inside the datamodel classes. The reason we need a plan is that we have certain technical challenges. For example, if we move the createDoiRequest inside the Publication, we should also inject a clock in the Publication with one or another way in order to be able to set the create/update instants in a controlled way.  

I am not sure however, if this lies in the scope of this bugfix.